src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -105,6 +106,16 @@ export const ARShowScreen = ({ navigation, route }) => {
         <Icon.NamedIcon name="camera" color={colors.darkText} size={normalize(30)} />
       </TouchableOpacity>
 
+      <TouchableOpacity
+        style={[styles.animationButton, styles.generalButtonStyle]}
+        onPress={() => setStartAnimation(!startAnimation)}
+      >
+        {startAnimation ? (
+          <Icon.NamedIcon name="stop" color={colors.primary} size={normalize(40)} />
```

Comment:
this button behave more like a "pause" button, right? animations get paused instead of completely stoped to the beginning. in this case let's use a "pause" icon here.

Reply:
I need to explore this with a more specific animation. The animation of the sample file we have is not very clear. I think this button will stop the animation. It doesn't make it pause.

Reply:
Yes it is different, I looked more carefully and the animation is just paused. That's why I changed the icon.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/578/commits/b6f83169a8c3647c64aa371c9815f19973fbfb81